lib/flux/app-state.ts

```diff
@@ -162,7 +143,7 @@ export const actionMap = new ActionMap({
               shareURL: '',
               publishURL: '',
               tags: ([] as T.TagName[]).concat(
-                state.tag ? state.tag.data.name : []
+                selectedTag ? selectedTag.data.name : []
```

Comment:
no need to do it here in this PR but I hope you're starting to scream "middleware!" when you see code like this 😄